Loop/Managers/ExtensionDataManager.swift

```diff
@@ -140,7 +152,7 @@ final class StatusExtensionDataManager {
 }
 
 
-extension StatusExtensionDataManager: CustomDebugStringConvertible {
+extension ExtensionDataManager: CustomDebugStringConvertible {
```

Comment:
[nit] I don't see the value in reducing the name to `ExtensionDataManager`. If this is needed, update line 158 accordingly.

Reply:
It now provides contexts for both the status extension, and the intents extension.

Reply:
Updating the other names, thanks.